### PR TITLE
Eliminate race condition 

### PR DESCRIPTION
**Pull request by cadecairos**
... where a project update occurring after a thumbnail tail would erase a project thumbnail URL

Fixes mozilla/webmaker-desktop#123

### CODE REVIEW

File: services/api/routes/authenticated.js [hunk 1]
```diff
@@ -169,10 +169,7 @@ var routes = [
           project: numericSchema
         },
         payload: {
-          title: Joi.string().optional(),
-          thumbnail: Joi.object().keys({
-            320: Joi.string().optional()
-          }).default({})
+          title: Joi.string().required()
```
Comment (jbuck) [inline services/api/routes/authenticated.js]: What happens if I send a payload with a thumbnail? Will it give me a validation error? All of the current versions of the Android app will send the thumbnail along

File: test/services/api/handlers/projects.js [hunk 2]
```diff
@@ -1135,29 +1135,6 @@ experiment('Project Handlers', function() {
       });
     });
 
-    test('update thumbnail (320) succeeds', function(done) {
```
Comment (jbuck) [inline test/services/api/handlers/projects.js]: What if you left these tests in, but just changed what the thumbnail is equal to? Or are these tests totally redundant now. Maybe just one test?

File: services/api/routes/authenticated.js (hunk 1 above)
Comment (cadecairos) [inline services/api/routes/authenticated.js]: yes, it'll 400

File: test/services/api/handlers/projects.js (hunk 2 above)
Comment (cadecairos) [inline test/services/api/handlers/projects.js]: these are all invalid, since we're not setting thumbnails on this route anymore.

**Comment by jbuck** [comment]
@cadecairos r+

**Pull request merged**